### PR TITLE
Added data for FahimFBA

### PR DESCRIPTION
**Pull request by FahimFBA**
Added data for my GitHub profile

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/284"></a>

### CODE REVIEW

File: public/data/FahimFBA.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/Fahim_FBA",
+        "icon": "twitter"
+      }
+    ]
+  }
+
```
Comment (Panquesito7) [inline public/data/FahimFBA.json:18]: Missing an endline.
```suggestion

```

File: public/data/FahimFBA.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Md. Fahim Bin Amin",
+    "bio": "Microsoft Research Investigation Contributor to OSS || Competitive Programmer || Tech-enthusiasts || Student of Computer Science and Engineering at United International University",
```
Comment (Panquesito7) [inline public/data/FahimFBA.json:3]:
```suggestion
    "bio": "Microsoft Research Investigation || Contributor to OSS || Competitive Programmer || Tech-enthusiasts || Student of Computer Science and Engineering at United International University",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/FahimFBA.json (hunk 2 above)
Comment (FahimFBA) [reply public/data/FahimFBA.json]: Can you please check that out now? I did the change afterward.

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**